### DISCUSSION TITLE
Can i create pixi3D project with svelte framwork?

### DISCUSSION THREAD

**Discussion by gilzero04** [jnsmalm/pixi3d#190] [General]
Hello, I test using pixijs in svelte framwork, it worked. Why I can't using pixi3D in svelte framwork? After i run dev show error same below.

![image](https://github.com/jnsmalm/pixi3d/assets/50231337/90f7261d-c4e1-44a4-8656-1613ef790f86)
![image](https://github.com/jnsmalm/pixi3d/assets/50231337/a1fdc89a-1727-4b6e-926b-6eec4593b2ef)

### My code
```Typescript
<script lang="ts">
    import { Application, Assets, Renderer } from "pixi.js";
    import { Model } from "pixi3d/pixi7";
    import { onMount } from 'svelte';

    onMount( async() =>{
        console.log("onMount");
    });
</script>

<h1>Welcome to SvelteKit</h1>
<p>Visit <a href="https://kit.svelte.dev">kit.svelte.dev</a> to read the documentation</p>
```

**Comment by jnsmalm** [comment]
Not sure, don't know much about Svelte. Could you make it work outside of Svelte?

**Reply by gilzero04** [reply]
Thank you, No problem. I can using nodejs.
